wsgidav.conf

```diff
@@ -45,7 +45,10 @@ ssl_private_key = "wsgidav/server/sample_bogo_server.key"
 
 # Add the MS-Author-Via Response Header to OPTIONS command to allow editing
 # with Microsoft Office (default: True)
-add_header_MS_Author_Via = True	
+add_header_MS_Author_Via = True
+
+# Add custom headers
+response_headers = [('Access-Control-Allow-Origin', 'http://example.org')]

```

Comment:
Should be commented out as example, but not enabled by default.
The file also uses double quotes throughout.